core/model/library.py

```diff
@@ -393,6 +394,7 @@ def estimated_holdings_by_language(self, include_open_access=True):
             .join(Work.presentation_edition)
             .filter(Edition.language != None)
             .group_by(Edition.language)
+            .order_by(desc(func.count(Work.id).label("work_count")))
```

Comment:
Is `order_by` necessary here? It seems like we pass this query directly into a `Counter` object, that doesn't assume order.

Reply:
I see what you mean.   I'll remove this.